constants.js

```diff
@@ -1,4 +1,4 @@
-let URL = 'http://calico.palisadoes.org/talawa/graphql';
+let URL = 'https://talawa-graphql-api.herokuapp.com/graphql';
```

Comment:
Please revert this. You are right in that it is incorrect, but we should limit the changes to fixing test coverage. Please open another issue to fix this and I will assign it to you.